tests_common/setup.py

```diff
@@ -19,7 +19,7 @@
 
 from setuptools import find_packages, setup
 
-version = "4.3.1"
+version = "4.3.1.1"
```

Comment:
I think this might conflict with the constraint on core (line 27). You'll need to put that to `inmanta-core~=4.3.1.dev` I think.

Reply:
Also, this could break `bumpversion` but I think we only ever run that on the dev branches so it should probably be safe. Can't say for sure at the moment.

Reply:
ow, thanks, good catch